components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -338,6 +286,22 @@ class AutoPushFeature(
         return (System.currentTimeMillis() - prefLastVerified) >= PERIODIC_INTERVAL_MILLISECONDS
     }
 
+    /**
+     * Observers that want to receive updates for new subscriptions and messages.
+     */
+    interface Observer {
+
+        /**
+         * A subscription for the scope is available.
+         */
+        fun onSubscriptionChanged(scope: String) = Unit
```

Comment:
Almost everywhere else `scope` is typed as `PushScope`, but here it's a String. Can you stick to one type throughout?

Reply:
The `typealias` was internal to just the `Connection` to avoid confusion between the `scope` and `appServerKey` since they're all strings.

I can expose that to consumers if it seems clearer.